lib/portage/util/env_update.py

```diff
@@ -294,6 +295,17 @@ def _env_update(makelinks, target_root, prev_mtimes, contents, env, writemsg_lev
         except IOError as e:
             if e.errno != errno.ENOENT:
                 raise
+    else:
+        etc_path = Path(eroot, "etc")
+        prelink_files = (
+            etc_path.joinpath("prelink.conf.d", "portage.conf"),
+            etc_path.joinpath("prelink.cache"),
+            etc_path.joinpath("prelink.conf"),
+            Path(eroot, "usr", "sbin", "prelink"),
```

Comment:
I think drop this bit -- the 'prelink' package (now last-rited) would uninstall this on removal.